Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/SwapShiftController.cs

```diff
@@ -997,6 +1005,9 @@ private async Task DeclineSwapShiftRequestAsync(string accessToken, string teams
             var httpClient = this.httpClientFactory.CreateClient("ShiftsAPI");
             httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", accessToken);
 
+            // Send Passthrough header to verify the sender of request in outbound call.
```

Comment:
For **all** of these comments:
Send Passthrough header to verify the sender of request in outbound call.

Replace "verify" with "indicate", as this is a hint, not a strong guarantee.